api/models/Medication.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const validate = require('validate.js');
+
+module.exports = {
+  attributes: {
+    medicationName: Sequelize.STRING,
+    medicationDosage: Sequelize.STRING,
+    medicationFrequency: Sequelize.STRING,
+    medicationStartDate: Sequelize.STRING,
```

Comment:
Future improvement: Make dates `Sequelize.DATE` in the future. One of the major reasons to jump to Sequelize over Sails's built-in Waterline was to get more data type options.

Right now, we're just parsing in the text, so this works for now. I'll create a Trello ticket to come back to this.